elasticsearch/build.gradle.kts

```
@@ -18,10 +18,11 @@ dependencies {
 
     implementation(mn.micronaut.jackson.databind)
 
-    testImplementation(mnTestResources.testcontainers.elasticsearch)
     testImplementation(mn.groovy.json)
     testImplementation(mnSecurity.micronaut.security)
     testImplementation(mn.reactor)
+    implementation(platform(mnTest.boms.testcontainers))
+    implementation(libs.testcontainers.elasticsearch)
```

```suggestion
    testImplementation(platform(mnTest.boms.testcontainers))
    testImplementation(libs.testcontainers.elasticsearch)
```

This is a big mistake @lissam1 we can not add test containers dependencies in the main classpath